### DISCUSSION TITLE
Welcome to Hackathon-1.0 Discussions!

### DISCUSSION THREAD

**Discussion by SatinderpalSingh** [Computer-Science-and-Engineering-GNDEC/Hackathon-1.0#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JASKAMAL22** [comment]
Hello @all. Welcoming you in the discussion section. I would be happy to collaborate with you all. Hope we together build a strong community :)
                          If WE as a community don't step up to help each other, then WHO will?

**Reply by SatinderpalSingh** [reply]
Welcome @JASKAMAL22 to community, Congratulation you are the first one to post in discussion forum. Introduce yourself and name of your team.

**Comment by Gurinder485** [comment]
Is Hackathon online or we have to be physically present in college campus

**Reply by SatinderpalSingh** [reply]
The Hackathon will be online, but you may use Computer Centre for sitting at day time. You need not to present physically in campus.

**Comment by aksh45** [comment]
Is there any accommodation available for dayscholars ?

**Reply by SatinderpalSingh** [reply]
No accommodation will be provided, as it is an online event.

**Reply by aksh45** [reply]
ok sir. Thanks for information.

**Comment by shivamjjha** [comment]
Please tell when will the topics provided. @SatinderpalSingh sir

**Reply by SatinderpalSingh** [reply]
The topics will be provided before starting time of hackathon

**Comment by palak199** [comment]
Are we supposed to choose from the problem statements that are given on the github page, or we can propose any idea (related to college) ourselves as well?
@SatinderpalSingh sir

**Reply by SatinderpalSingh** [reply]
You have to choose the problems from the given statements.

**Comment by Gurinder485** [comment]
On which email id we have to mail to get linux host name

**Reply by SatinderpalSingh** [reply]
cc@gndec.ac.in

**Comment by PriyankaJhamb** [comment]
Sir, On which repository, we have to send pull request for solution?

**Reply by SatinderpalSingh** [reply]
You have to send PR at projects branch

**Comment by SatinderpalSingh** [comment]
@all you need to make a progress.md file in the root folder of the project.
 
Following will be the structure of the md file.

| Hours since 12:00 noon of 5th of March 21 | Progess |
| 1 | Studied requirements, searched for exiting software, frame work, selected X software |
| 2 | Software X installed and working fine |
| 3 | Feature Y added, check it in action at: <URL> |

**Comment by SuhelMehta9** [comment]
Here is our hosted mail server  
[Admin](https://surajdadral.me/admin)  
[User mails](https://surajdadral.me/SOGo)  

Username and password for accessing the mailbox  

username: test@mail.surajdadral.me  
password: test@123

**Comment by PriyankaJhamb** [comment]
Sir,
Sorry, I have not seen this email till now. So, our team has not uploaded
progress.md file.
But we have included all things in Readme.md file.

It is very kind of you if you will accept it as progress report.

Thanks & Regards
Priyanka Jhamb

**Reply by PriyankaJhamb** [reply]
Sir, Can we make or rename progress.md file now and send pull request?

**Comment by prabhkaur301** [comment]
Team Falcon-Coders
About Project:-https://github.com/Falcon-Coders/Hackathon-1.0/blob/projects/README.md
Server-link : https://46867a9007f2.ngrok.io

To enter into server ,login credentials are required.
So we have send login credentials at cc@gndec.ac.in

Team leader -Priyanka Jhamb
email-priyankajhamb73@gmail.com

**Comment by SatinderpalSingh** [comment]
You are suppose to post URLs at https://github.com/Computer-Science-and-Engineering-GNDEC/Hackathon-1.0/discussions/59 this thread is for general discussions.

**Comment by PriyankaJhamb** [comment]
Ok Sir

**Comment by SatinderpalSingh** [comment]
The presentations regarding the projects will be stated at 9AM in online mode. The time slot given to every team will be 15 mins. Book your slot by entering your details in this [sheet.](https://docs.google.com/spreadsheets/d/1l3QxWPWqkOeKFk9CwiJql5c1B-qHqESWCsNo2jew3BA/edit)

**Reply by SatinderpalSingh** [reply]
https://computer-science-and-engineering-gndec.github.io/Hackathon-1.0/end.html
